CHANGES.md

```diff
@@ -1,3 +1,20 @@
+## edge-19.7.1
+
+* CLI
+  * Added more descriptive output to the `linkerd check` for control plane
+    ReplicaSet readiness
+  * Renamed `config.linkerd.io/debug` annotation to
+    `config.linkerd.io/enable-debug-sidecar`, to match the CLI flag that sets it
```

Comment:
Is this a breaking change? In the past we've doing something like:

```
  * **Breaking Change:** Renamed the `config.linkerd.io/debug` annotation to...
```

Reply:
Ah, yeah, good call -- @ihcsim, is this breaking?

Reply:
Yes, it is. Thanks for calling it out.

Reply:
cool, updated!